lib/main_publiccloud.pm

```diff
@@ -0,0 +1,118 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: module loader of container tests
+# Maintainer: qa-c@suse.de
+
+package main_publiccloud;
+use Mojo::Base 'Exporter';
+use utils;
+use version_utils;
+use main_common qw(loadtest load_extra_tests_prepare);
+use testapi qw(check_var get_var);
+
+our @EXPORT = qw(
+  load_publiccloud_tests
+);
+
+sub load_podman_tests() {
+    loadtest 'containers/podman';
+    loadtest 'containers/podman_3rd_party_images';
+}
+
+sub load_docker_tests() {
+    loadtest 'containers/docker';
+    loadtest 'containers/docker_runc';
+    loadtest 'containers/docker_image';
+    loadtest 'containers/docker_3rd_party_images';
+    loadtest 'containers/registry';
+    loadtest 'containers/zypper_docker';
+}
+
+sub load_qam_publiccloud_tests {
```

Comment:
Is it a good time to rename `qam` by `maintenance`? I think it reflects better what it is and we eliminate the separation between the "old" departments.
`load_maintenance_publiccloud_tests`

Reply:
*this change is ready for review*